frontend/src/components/Course/styled.ts

```diff
@@ -227,9 +227,8 @@ export const CourseDescription = styled('p')<InfoContainerTypes>(
 
 export const InfoContainer = styled(Box)<InfoContainerTypes>(({ type }) => ({
   display: 'flex',
-  height: '30px',
-  alignItems: 'end !important',
-  alignSelf: 'end !important',
+  alignItems: 'start !important',
```

Comment:
Please, use styled method from MUI instead styled from styled-component library. Its helps to avoid !important

Reply:
https://mui.com/guides/interoperability/#styled-components